ir/memory.cpp

```diff
@@ -1009,24 +1009,26 @@ static expr mk_liveness_array() {
   return expr::mkInt(-1, num_nonlocals);
 }
 
-void Memory::mk_nonlocal_val_axioms(bool skip_consts) {
+void Memory::mkNonlocalValAxiom(unsigned i) {
```

Comment:
This API change is not good IMHO. Especially for the new memory encoding that doesn't use multiple blocks.

Reply:
Okay, I'll change the API.